src/main/java/com/fvv/bookstore/util/FileUtil.java

```diff
@@ -0,0 +1,72 @@
+package com.fvv.bookstore.util;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+/**
+ * Class to save and read a file.
+ * 
+ * @author Fatima Vasquez
+ * <p>Created on 2018</p>
+ * @version 1.0 	
+ *
+ */
+public class FileUtil {
+
+	/**
+	 * To create an audit file.
```

Comment:
you are not creating an auditing file, you are adding a entry to a audit file. write a better javadoc

Reply:
ok